### ISSUE TITLE
WOF Neighborhood missing admin info

### ISSUE THREAD

**Issue by tigerlily-he** [pelias/pelias#615]
Hey team! 

I was using your awesome geocoding engine when I noticed something interesting.
Let me tell you more about it.

---
#### Here's what I did :innocent:

I was searching for a [neighborhood in Cambodia called Tuk Laak](http://pelias.github.io/compare/#/v1/search%3Ftext=Tuk%20Laak) [(whosonfirst:neighbourhood:85902737)](https://whosonfirst.mapzen.com/spelunker/id/85902737/)

The hierarchy should be: 
```
the continent of Asia
the country of Cambodia
the region of Phnom Penh
the county of Prampir Meakkakra
```

---
#### Here's what I got :scream_cat:

```
"features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          104.911622,
          11.568929
        ]
      },
      "properties": {
        "id": "85902737",
        "gid": "whosonfirst:neighbourhood:85902737",
        "layer": "neighbourhood",
        "source": "whosonfirst",
        "source_id": "85902737",
        "name": "Tuk Laak",
        "confidence": 1,
        "match_type": "exact",
        "accuracy": "centroid",
        "label": "Tuk Laak"
      },
      "bbox": [
        104.868234863,
        11.5426893834,
        104.928441448,
        11.5782521331
      ]
    }
  ],
```

---
#### Here's what I was expecting :sparkles:

There is no admin info (county, region, country). Hence ["Tuk Laak, Cambodia"](http://pelias.github.io/compare/#/v1/search%3Ftext=Tuk%20Laak,%20Cambodia) falls back to just Cambodia

---
#### Here's what I think could be improved :trophy:

For some reason the WOF hierarchy wasn't imported properly

**Comment by orangejulius** [comment]
Ouch. If the hierarchy is correct in the spelunker then that means its a bug on our side somewhere in https://github.com/pelias/wof-admin-lookup

**Comment by tigerlily-he** [comment]
Same with [Toul Tapeung, Cambodia](http://pelias.github.io/compare/#/v1/place%3Fids=whosonfirst:neighbourhood:85902735) (whosonfirst:neighbourhood:85902735), although there is also a record for a different spelling [Tuŏl Tumpung, Cambodia](http://pelias.github.io/compare/#/v1/place%3Fids=whosonfirst:neighbourhood:421181467) that _does_ include admin data

**Comment by missinglink** [comment]
this might be related to the work done in https://github.com/whosonfirst-data/whosonfirst-data/pull/660 which landed 6 days ago

**Comment by missinglink** [comment]
this issue should be fixed in due course by new `elasticsearch` and `placeholder` builds without further action.

**Comment by dianashk** [comment]
I don't think it will be fixed by the upcoming placeholder integration. We are already using placeholder for admin only queries in production and it's not working there. Also on dev we already have Stephen's latest placeholder integration work and we get the same results there. It's something else.

**Comment by dianashk** [comment]
Nevermind! I see you meant that it would get refreshed with the latest builds since the data has been fixed at the source. That's great!
